fvm/fvm_test.go

```diff
@@ -1432,6 +1433,7 @@ func TestSignatureVerification(t *testing.T) {
 						view state.View,
 						programs *programs.Programs,
 					) {
+						t.Skip("temporarily disabled")
```

Comment:
Instead of skipping for all pairs, can we only skip for the excluded case(s) above?

Reply:
This skip is no longer needed, since the change in VerifySignature was reverted.